cfg/config.go

```diff
@@ -156,3 +160,21 @@ func (c Configuration) GetGlobalDefaultResources() (res corev1.ResourceRequireme
 func GetGlobalRepository() string {
 	return fmt.Sprintf("s3:%s/%s", Config.GlobalS3Endpoint, Config.GlobalS3Bucket)
 }
+
+// GetGlobalFailedJobsHistoryLimit returns the global failed jobs history limit.
+// Returns global KeepJobs if unspecified.
+func (c Configuration) GetGlobalFailedJobsHistoryLimit() int {
+	if c.GlobalFailedJobsHistoryLimit != -1 {
```

Comment:
If the input is less that `-1` then we get a negative number.

```suggestion
	if c.GlobalFailedJobsHistoryLimit > -1 {
```